draft-ietf-ppm-dap.md

```diff
@@ -1243,10 +1240,14 @@ Each of these messages is constructed as follows:
   * `report_share.encrypted_input_share` is the intended recipient's (i.e.
     Helper's) encrypted input share.
 
-  * `payload` is set to the `outbound` message computed by the previous step.
+  * `ping_pong_message` is set to the `outbound` message computed by the
+     previous step.
 
-Next, the Leader creates an `AggregationJobInitReq` message structured as
-follows:
+It is not possible for `state` to be of type `Finished` during Leader
+initialization.
```

Comment:
IMO this doesn't need to be said.

Reply:
I dunno, we're telling DAP authors here to go look at the `State` enum in VDAF and how to deal with its `Continued` and `Rejected` variants, so they'll quite naturally wonder what they should do about `Finished`, won't they?

Reply:
Yeah fair enough.